lib/custom_linters/translations/view_translations_linter.rb

```diff
@@ -94,7 +94,7 @@ def potential_substrings(stringified_view)
       # removes leading and ending extra blankspace, and blank strings, and downcase, returning an array like:
       # ["list all attributes", "success"]
       #  Further reading: https://stackoverflow.com/a/54405741/5331859
-      potential_substrings_no_html_tags = ActionView::Base.full_sanitizer.sanitize(stringified_view).split("\n")
+      potential_substrings_no_html_tags = ActionView::Base.full_sanitizer.sanitize(stringified_view.gsub(/<%.*?%>/, "")).split("\n")
```

Comment:
we have some emails on db/seedfiles/translations/en/dc/user_mailer.rb, that has a lot of HTML tags, if I'm not mistaken this will wipe out all the format, did we validate that this will work?

Reply:
Great question, @TreyE 

Though does appear to be targeting ERB tags specifically, with the regular expression rather than all HTML tags. I just checked the file in question (`user_mailer.rb`) and it appears to not have any ERB tags in the file. 

regex examples: https://rubular.com/r/hFBMSR7rYzEmBR

Reply:
This is targeting to remove ERB tags specifically.  This is appropriate here (and only here) as this portion of the checker only looks for missing translations that live outside of an ERB tag.  Check the full method and you'll see that the portions of the checker which look for missing translations *inside* the ERB tags remain unchanged.

The reason I had to add in this substitution is because of a change to how Nokogiri works between versions 1.10 and 1.13:
1. ActionView::Base.full_sanitizer actually delegates, behind the scenes to Nokogiri to scrub html tags
2. Nokogiri 1.10, when scrubbing HTML, will remove ERB tags, code and content.
3. At some point between 1.10 and 1.13, it stopped removing ERB tags and content.
4. So when you upgrade Nokogiri, ERB tags are no longer pruned in this portion of the translation checker, and you start to get a whole bunch of false positives.